crypto/keyring/keyring_ledger_test.go

```diff
@@ -1,6 +1,6 @@
 //+build ledger test_ledger_mock
 
-package keyring
+package keyring_test
```

Comment:
@amaurym  Although I updated package name to keyring_test, the cycle dep issue still persists when I run either file tests or package tests. `encCfg := simapp.MakeTestEncodingConfig()` causes this issue.
```
package github.com/cosmos/cosmos-sdk/crypto/keyring
	imports github.com/cosmos/cosmos-sdk/simapp/params
	imports github.com/cosmos/cosmos-sdk/client
	imports github.com/cosmos/cosmos-sdk/client/flags
	imports github.com/cosmos/cosmos-sdk/crypto/keyring: import cycle not allowed
```

Reply:
the import `github.com/cosmos/cosmos-sdk/crypto/keyring -> imports github.com/cosmos/cosmos-sdk/simapp/params` is incorrect, could you try to figure out where it happens, and try to remove it?